packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/Task/Task.js

```diff
@@ -60,7 +60,12 @@ function Task ({
   }
 
   return (
-    <Box key={annotation.id} basis='auto'>
+    <Box
+      key={annotation.id}
+      as='fieldset'
```

Comment:
This will mark up all tasks as fieldsets, not just survey tasks. That's likely to be a problem for some tasks eg. #2578 the transcription task instructions aren't valid fieldset legends.

Reply:
I'm thinking that `fieldset` must have `legend`, so the legend should also be defined in `Task.js`. Otherwise, it's too easy to add a new task without a legend.

Alternatively, move the `fieldset` down to the survey task itself, so that individual tasks are responsible for how they are marked up.